### DISCUSSION TITLE
stg_dbt__model_executions compiled version throwing error

### DISCUSSION THREAD

**Discussion by calkan-cue** [brooklyn-data/dbt_artifacts#318] [Q&A]
Hello I am trying to use the 
dbt artifacts 2.3.0 

when the 
**stg_dbt__model_executions.sql** 
is compiled 

> with base as (
>     select *
>     from {{ ref('model_executions') }}
> ),
> enhanced as (
>     select
>         {{ dbt_artifacts.surrogate_key(['command_invocation_id', 'node_id']) }} as model_execution_id,
>         command_invocation_id,
>         node_id,
>         run_started_at,
>         was_full_refresh,
>         {{ split_part('thread_id', "'-'", 2) }} as thread_id,
>         status,
>         compile_started_at,
>         query_completed_at,
>         total_node_runtime,
>         rows_affected,
>         {% if target.type == 'bigquery' %}
>             bytes_processed,
>         {% endif %}
>         materialization,
>         schema, -- noqa
>         name,
>         alias
>     from base
> )
> select * from enhanced

**stg_dbt__model_executions.sql** 
> create or replace view `dlake`.`dbt_dbt_artifacts_staging`.`stg_dbt__model_executions`
> OPTIONS(
> description="""Staging model that contains data about model executions. One row per model execution."""
> )
> as with base as (
> select *
> from `dlake`.`dbt_dbt_artifacts_source`.`model_executions`
> ),
> enhanced as (
> select
to_base64(md5(CAST(coalesce(cast(command_invocation_id as STRING), '') || '-' || coalesce(cast(node_id as STRING), '') as STRING))) as coalesce(cast(command_invocation_id as > STRING), '') || '-' || coalesce(cast(node_id as STRING), '')_hash
> as model_execution_id,
> command_invocation_id,
> node_id,
> run_started_at,
> was_full_refresh,
> split(
> thread_id,
> '-'
> )[safe_offset(1)]
> as thread_id,
> status,
> compile_started_at,
> query_completed_at,
> total_node_runtime,
> rows_affected,
> bytes_processed,
> materialization,
> schema, -- noqa
> name,
> alias,
> message
> from base
> )
> select * from enhanced;

This is compiled result but it has error even though it is coming from package itself. 

> 11:57:05  Database Error in model stg_dbt__model_executions (models/staging/stg_dbt__model_executions.sql)
> 11:57:05    Syntax error: Expected ")" but got "(" at [19:148]
> 11:57:05    compiled Code at target/run/dbt_artifacts/models/staging/stg_dbt__model_executions.sql
> 11:57:05  
> 11:57:05  on-run-end failed, error:
> 11:57:05   Syntax error: Unclosed string literal at [35:13]

I stuck here. 
Those are the dependencies I have:

packages.yml
`
packages:
  - package: dbt-labs/codegen
    version: 0.9.0
  - package: dbt-labs/dbt_project_evaluator
    version: 0.4.1
  - package: dbt-labs/dbt_external_tables
    version: 0.8.2
  - package: brooklyn-data/dbt_artifacts
    version: 2.3.0
  - package: fivetran/shopify
    version: [">=0.8.0", "<0.9.0"]
  - package: fivetran/ad_reporting
    version: [">=1.3.0", "<1.4.0"]
  - package: fivetran/netsuite
    version: [">=0.7.0", "<0.8.0"]
  - package: fivetran/salesforce
    version: [">=0.8.0", "<0.9.0"]
  - package: dbt-labs/metrics
    version: [">=1.4.0", "<1.5.0"]
`

requirements.txt
`
dbt-bigquery==1.4.1   
dbt-core==1.4.5  
dbt-extractor==0.4.1  
`

**Comment by jared-rimmer** [comment]
Hi @calkan-cue are you able to open this as an issue please?
